Lib9c/Policy/MaxBlockBytesPolicy.cs

```diff
@@ -34,12 +34,12 @@ private MaxBlockBytesPolicy(int defaultValue)
                     value: 1024 * 100))         // 100 KiB
                 // Note: Temporary limit increase for resolving
                 // https://github.com/planetarium/NineChronicles/issues/777.
-                // Issued for v100081.
-                // FIXME: Starting index must be finalized accordingly before deployment.
+                // Issued for v100081.  Temporary ad hoc increase was introduced
+                // around 2_500_000.
                 .Add(new SpannedSubPolicy<int>(
                     startIndex: 2_000_001,
                     value: 1024 * 1024 * 10))    // 10 MiB
-                // Note: Reverting back to the previous limit.  Issued for v100084.
+                // Note: Reverting back to the previous limit.  Issued for v100086.
                 // FIXME: Starting index must be finalized accordingly before deployment.
                 .Add(new SpannedSubPolicy<int>(
                     startIndex: 3_000_001,
```

Comment:
9c is taking about 6,600-7,200 blocks a day, and as of November 10, it is about 2.72 million blocks, so I think it would be conservative to generate this much if the limit is not urgent.

```suggestion
                    startIndex: 2_800_001,
```